profiles/base.yaml

```diff
@@ -817,8 +817,12 @@ consumable_lockboxes:
 
 # pick.lic settings for live boxes
 # https://elanthipedia.play.net/Lich_script_repository#pick
+# Set to false if you want to pick all boxes ignoring exp mind states.
 stop_pick_on_mindlock: true
-pick_blind: false
```

Comment:
Removing `pick_blind` from `base.yaml` means a bunch of YAMLs checked into `/profiles` suddenly become invalid. That's why the TravisCI build is failing.

Reply:
Could the test script be updated in this PR to handle the renamed setting?

`dr-scripts/test/test_valid_yaml.rb:32`